.github/workflows/lint.yml

```diff
@@ -12,5 +10,7 @@ jobs:
       - uses: actions/setup-node@v3
         with:
           node-version-file: ".node-version"
```

Comment:
Credit to @reefdog for this suggestion in [the previous pull request](https://github.com/PhilanthropyDataCommons/data-viewer/pull/7#discussion_r1117785046)!

> I prefer exact versions in the codebase for maximum reproducibility and traceability.

I completely agree when it comes to dependencies, but for Node.js versions I tend to come down on only pinning to a major version: they have a frequent release cadence, and have (at least in my experience) done an excellent job of following semver - I don't think I've ever had a broken build due to a new Node.js version!

Reply:
I would say node is still a dependency here while I acknowledge it is a more fundamental dependency rather than what we typically call dependencies. A key difference between this project and the `service` project is that it is only a build-time dependency and not a runtime dependency. That difference plus deference to the people doing the bulk of the work in a repo leads me to not object to the change here while I would still object to undoing the specificity in the `service` project.